pkg/frontend/openshiftcluster_putorpatch.go

```diff
@@ -41,18 +43,20 @@ func (f *frontend) putOrPatchOpenShiftCluster(w http.ResponseWriter, r *http.Req
 	subId := chi.URLParam(r, "subscriptionId")
 	resourceProviderNamespace := chi.URLParam(r, "resourceProviderNamespace")
 
+	identityURL := r.Header.Get("x-ms-identity-url")
```

Comment:
I'll add a todo in msi dataplane repo to make this an exportable const: https://github.com/Azure/msi-dataplane/blob/557894f3863f5591445903674cc6ba90e2a26fb7/pkg/dataplane/constants.go#L13